regulations/templates/regulations/preamble-partial.html

```diff
@@ -1,9 +1,9 @@
 {% load macros render_nested %}
 
 <section id="{{full_id}}" class="reg-section" data-page-type="preamble-section" data-doc-id="{{doc_number}}">
-  <ul id="preamble-read">
+  <div id="preamble-read">
     {% render_nested template=sub_template context=sub_context %}
-  </ul>
+  </div>
```

Comment:
When I changed this I was looking at CFR sub_context and saw it was returning `<section>` and `<ol>`s in them, not realizing the regular preamble was returning the list items... I'll change it back and instead wrap CFR subtext in a list item instead. 😑